services/middlewareServices/FormSGService.ts

```diff
@@ -0,0 +1,57 @@
+import FormSG from "@opengovsg/formsg-sdk"
+import { Request, Response, NextFunction } from "express"
+
+import logger from "@logger/logger"
+
+import { AuthError } from "@root/errors/AuthError"
+
+const formsg = FormSG()
+
+export default class FormSGService {
```

Comment:
we should probably rename this to indicate its purpose - something like `FormsVerificationService` as its main function is essentially that?

Reply:
idt the `class` is required; this might be better off being an object

Reply:
Renamed in https://github.com/isomerpages/isomercms-backend/pull/458/commits/16ee9e43f759785cfc9f22a767998d851e292077. As discussed offline, we will maintain this format for consistency

Reply:
nit: could we call this `FormSgProcessingService` or `FormsProcessingService`? Apologies for this but FormSG in a name just looks weird ;--;

Reply:
db85651